src/psyclone/domain/lfric/kern_call_acc_arg_list.py

```diff
@@ -63,10 +95,10 @@ def field_vector(self, argvect, var_accesses=None):
 
         :param argvect: the field vector to add.
         :type argvect: :py:class:`psyclone.dynamo0p3.DynKernelArgument`
-        :param var_accesses: optional VariablesAccessInfo instance to store \
+        :param var_accesses: optional VariablesAccessInfo instance to store
             the information about variable accesses.
-        :type var_accesses: \
-            :py:class:`psyclone.core.VariablesAccessInfo`
+        :type var_accesses: Optional[
```

Comment:
As you have kindly added optional to this method, could you add the same to the other existing methods.

Reply:
Done.

Reply:
Thanks, but there is still one missing in the `stencil` method

Reply:
Oops.